### ISSUE TITLE
Button hotkey priority is always active when grid is showing

### ISSUE THREAD

**Issue by raysmith59** [brittyazel/Neuron#273]
If there isn't a spell active on the button it shouldn't activate in this scenario.  This was fixed a while back for hidden grid but I'm just noticing it's also happening with grid showing.  See #246 for reference.

**Comment by brittyazel** [comment]
Yeah this is actually how it has been designed. The fact that priority binds would become disabled upon not showing the grid was a bi-product of hiding those frames, and them becoming essentially innert. I'll have to figure out how to override this, but as the system was originally built without this in mind it could take a bit

**Comment by raysmith59** [comment]
Yeah, no worries in terms of time, now that the buttons are showing properly with grids disabled this is no longer an issue for me so it's just a generic bug that probably nobody is experiencing.
